runtime/hydradx/Cargo.toml

```diff
@@ -154,7 +155,6 @@ runtime-benchmarks = [
     "cumulus-pallet-xcmp-queue/runtime-benchmarks",
     "pallet-uniques/runtime-benchmarks",
     "pallet-omnipool/runtime-benchmarks",
-    "pallet-transaction-pause/runtime-benchmarks",
```

Comment:
why removed?

Reply:
cause it was duplicated. If you check the file further up you find "pallet-transaction-pause/runtime-benchmarks"